src/common/dsp/oscillators/WavetableOscillator.cpp

```diff
@@ -62,10 +62,12 @@ void WavetableOscillator::init(float pitch, bool is_display, bool nonzero_init_d
 
     n_unison = limit_range(oscdata->p[wt_unison_voices].val.i, 1, MAX_UNISON);
 
+    isOneShot = 0;
```

Comment:
Since this is a bool you can use the constant true and false for better readability here

Reply:
isOneShot is actually not used anymore. I added it because it was used as a check in convolute(), but I realised later that it wasn't needed. I forgot to remove it. 
Should I remove it or just change it to a bool?

Reply:
If it is completely unused it can be removed.